portfolio/src/main/webapp/script.js

```diff
@@ -65,11 +71,11 @@ const getComments = () => {
 * @param {Array<Object>} comments - Array of comments retrieved from servlet
 * @param {Element} commentContainerElement - the target HTML element that will contain the comments.
 */
-const appendComments = (comments, commentContainerElement) => {    
+const appendComments = (comments, commentContainerElement) => {
   Object.keys(comments).forEach(commentId => {
     // Create each element with its properties
```

Comment:
You can save comments[commentId] into a variable to avoid typing it over and over again:

const comment = comments[commentId];

Then you can use "comment" instead of "comments[commentId]". This simplifies things and improves readability.

Reply:
Sure, thanks for the recommendation.